pkgs/development/interpreters/bats/libraries.nix

```diff
@@ -47,6 +47,29 @@
     };
   };
 
+  bats-detik = stdenv.mkDerivation rec {
+    pname = "bats-detik";
+    version = "1.2.1";
+    src = fetchFromGitHub {
+      owner = "bats-core";
+      repo = "bats-detik";
+      rev = "v${version}";
+      sha256 = "sha256-2BEIqRSc21oPjd9BgTLg5mGyAdNJYA2b7gZe7Nj2dks=";
```

Comment:
Prefer `hash = ...` to `sha256 = ...` Both currently work, but the former will make it easier if the algorithm ever needs to change.

Since there could be different hashing algorithms in use, this make sense.

Reply:
fixed sorry